extension/src/experiments/model/quickPick.test.ts

```diff
@@ -147,7 +147,7 @@ describe('pickExperiments', () => {
           description: '[exp-456]',
           detail: `Created:${formatDate(
             mockedExperiments[1].Created as string
-          )}, split:2.2000e+9, data/data.xml:22a1a29`,
```

Comment:
Spotted this bug that was happening with long integers while working on the refactoring.

Reply:
This was actually intended (see https://github.com/iterative/vscode-dvc/pull/2813#discussion_r1029869455). 

There is a long tail of edge cases here. For example, why would we treat `2200043556.1` differently? What if we had `10000000000000000`? The JS numbers rabbit hole is deep.

Reply:
Oh, whoops! Must have missed that comment! The table has the full number so I assumed it was a bug. Will revert this change :)

Reply:
Do you think we should update the webview?

Reply:
> Do you think we should update the webview?

Sure! Apologies if that's what you wanted originally, I misunderstood. When you say webview, are you referring to just the tooltips or the table as well? I'll open a pr that does both for now.

Reply:
I didn't even think about that until you replied. No need to apologise. It was probably me that missed something back in the original PR.